MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -824,6 +824,22 @@ private async Task<string> ExtractImageInternal(string inputPath, string contain
                 options.EnableTonemapping = false;
             }
 
+            if (imageStream.Width is not null && imageStream.Height is not null)
+            {
+                // For hardware trickplay encoders, we need to re-calculate the size because they used fixed scale dimensions
+                var darParts = imageStream.AspectRatio.Split(":");
+                var (wa, ha) = (int.Parse(darParts[0], CultureInfo.InvariantCulture), int.Parse(darParts[1], CultureInfo.InvariantCulture));
```

Comment:
This assumes both parts of the aspect ratio exist, and that they are both integers.
What happens if the aspect ratio is something like `2.40:1`?

Reply:
As long as this string is not empty it should contain both parts as the probe result is already sanitized in most cases. Let's see if there is really weird videos like this.